### PR TITLE
Move instructions on naming restrictions to Keys section

### PR DESCRIPTION
**Pull request by pradyunsg**
CF: https://github.com/toml-lang/toml/issues/499#issuecomment-388241800

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,6 +191,28 @@ In JSON land, that would give you the following structure:
 Whitespace around dot-separated parts is ignored, however, best practice is to
 not use any extraneous whitespace.
 
+Defining a key multiple times is invalid.
+
+```
+# DO NOT DO THIS
+name = "Tom"
+name = "Pradyun"
```
Comment (pradyunsg) [inline README.md]: I added my name to the document. Hopefully, nobody minds. :')

Comment (mojombo) [reply README.md]: Perfectly fine for such a great contributor!

Comment (pradyunsg) [reply README.md]: Yay! Thanks!

**Comment by pradyunsg** [comment]
I get to merge this. ✨

**Pull request merged**